calico_kubernetes/calico_kubernetes.py

```diff
@@ -894,6 +911,9 @@ def read_config_file():
         DEFAULT_POLICY_VAR: "allow",
         KUBE_AUTH_TOKEN_VAR: None,
         LOG_LEVEL_VAR: "INFO",
+        KUBE_CLIENT_CERTIFICATE_VAR :None,
```

Comment:
formatting nitpick: Make sure you've got consistent spacing around your `:`

e.g
```
KUBE_CLIENT_CERTIFICATE_VAR: None,
```
rather than
```
KUBE_CLIENT_CERTIFICATE_VAR :None,
```